src/curses/ui_call_flow.c

```diff
@@ -1389,9 +1389,11 @@ call_flow_column_add(ui_t *ui, const char *callid, address_t addr)
     vector_append(column->callids, (void*)callid);
     column->addr = addr;
     if (setting_enabled(SETTING_ALIAS_PORT)) {
-        strcpy(column->alias, get_alias_value_vs_port(addr.ip, addr.port));
+        strncpy(column->alias, get_alias_value_vs_port(addr.ip, addr.port), sizeof(column->alias) - 1);
+        column->alias[sizeof(column->alias) - 1] = '\0';
```

Comment:
Maybe we could create a sng_strncpy function in utils.c that does this two lines and reuse it everywhere

```
char * sng_strncpy(char *destination, const char *source, size_t num) {
    strncpy(destination, source, num - 1);
    destination[num - 1] = '\0';
}
```

We could also add validations to source and destination in the function